main.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'app'
+
+def show_menu
+  puts 'Please choose an option by entering a number:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person id'
+  puts '7. Exit'
```

Comment:
Nice job so far👍.  Currently, all prompts from 1 - 6, when entered, return an `undefined method` error 
<img width="512" alt="image" src="https://github.com/KAYEMBAJONATHAN/school-library/assets/64643995/bb82ac9c-c2c3-4aec-b1a3-0b4748cc75c0">


This is because these methods haven't been defined in your `app.rb` file as required😉. 
<img width="748" alt="image" src="https://github.com/KAYEMBAJONATHAN/school-library/assets/64643995/47a6c801-dc07-4ac2-bf57-c41a64ff07ac">

Kindly define and implement the necessary methods for each prompt so your console app can work properly😉.